tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExtensionsMetadataGeneratorTests.cs

```diff
@@ -36,5 +36,16 @@ public void GenerateExtensionsJson_Succeeds()
             Assert.Equal("BarExtension", extensions[1]["name"]);
             Assert.Equal(typeof(BarWebJobsStartup).AssemblyQualifiedName, extensions[1]["typeName"]);
         }
+
+        [Theory]
+        [InlineData("System.Linq.dll", true)]
+        [InlineData("Microsoft.Azure.WebJobs.Extensions.dll", true)]
+        [InlineData("Microsoft.Azure.WebJobs.Extensions.Http.dll", true)]
+        [InlineData("Microsoft.Azure.EventGrid.dll", false)]
+        [InlineData("MyCoolExtension.dll", false)]
```

Comment:
Maybe add some casing variation here to cover the scenario mentioned in my comment.